components/brave_shields/browser/brave_shields_util.cc

```diff
@@ -505,6 +508,24 @@ ControlType GetCookieControlType(
   return ControlType::BLOCK;
 }
 
+CookiesState GetOverallCookiesState(
+    HostContentSettingsMap* map,
+    content_settings::CookieSettings* cookie_settings,
+    const GURL& url) {
+  auto result = CookieRules::Get(map, url, ContentSettingsType::COOKIES);
```

Comment:
why don't we just check the pref for the global default? Or maybe I'm misunderstanding what's going on here?

Reply:
sorry, actually it's CookieSettings:: GetCookieSetting that we should be using here I think